index.bs

```diff
@@ -2428,6 +2435,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
                     [=client extension processing=] algorithm to create the [=client extension outputs=], for each
                     [=client extension=] in <code>|options|.{{PublicKeyCredentialRequestOptions/extensions}}</code>.
 
+            1. If |credentialIdFilter| [=list/is not empty=] and |credentialIdFilter| does not contain an item whose
+                {{PublicKeyCredentialDescriptor/id}}'s value is set to the value of [=credentialIdResult=], [=continue=].
```

Comment:
This should be prefixed with _assertionCreationData_, and I think this could read slightly better this way:

```suggestion
                {{PublicKeyCredentialDescriptor/id}}'s value equals <code>|assertionCreationData|.[=credentialIdResult=]</code>, [=continue=].
```